src/components/Grunnlag/Grunnlag.tsx

```diff
@@ -69,8 +61,15 @@ export function Grunnlag({ tidligstMuligUttak }: Props) {
         <Utenlandsopphold />
         <Alderspensjon alderspensjon={alderspensjon} />
         <AFP />
-        {isLoading && <SectionSkeleton />}
-        {isSuccess && <Pensjonsavtaler pensjonsavtaler={pensjonsavtaler} />}
+
+        {showLoader ? (
+          <SectionSkeleton />
+        ) : (
```

Comment:
Denne kan `Pensjonsavtaler` overta ansvaret for å kondisjonelt rendre avhengig av responsen fra pensjonsavtaler-kallet

Reply:
Jeg fant ikke noe spesifikasjon for denne SectionSkeleton komponent og hvordan loading egentlig skal vises. Når det gjelder hvilken komponent/container har ansvaret for logikken er det relatert til din første kommentar, se svar der.

Reply:
Avklart med Cecilie. Det skal ikke være noe Skeleton, men en spinner. Har kodet den i SectionHeader.